cmd/template/dbdriver/files/service/mysql.tmpl

```diff
@@ -44,16 +45,52 @@ func New() Service {
 	return s
 }
 
+// Health checks the health of the database connection by pinging the database.
+// It returns a map with keys indicating various health statistics.
 func (s *service) Health() map[string]string {
 	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
 	defer cancel()
 
+	stats := make(map[string]string)
+
+	// Ping the database
 	err := s.db.PingContext(ctx)
 	if err != nil {
-		log.Fatalf(fmt.Sprintf("db down: %v", err))
+		stats["status"] = "down"
+		stats["error"] = fmt.Sprintf("db down: %v", err)
+		log.Fatalf(fmt.Sprintf("db down: %v", err)) // Log the error and terminate the program
+		return stats
 	}
 
-	return map[string]string{
-		"message": "It's healthy",
+	// Database is up, add more statistics
+	stats["status"] = "up"
+	stats["message"] = "It's healthy"
+
+	// Get database stats (like open connections, in use, idle, etc.)
+	dbStats := s.db.Stats()
+	stats["open_connections"] = strconv.Itoa(dbStats.OpenConnections)
+	stats["in_use"] = strconv.Itoa(dbStats.InUse)
+	stats["idle"] = strconv.Itoa(dbStats.Idle)
+	stats["wait_count"] = strconv.FormatInt(dbStats.WaitCount, 10)
+	stats["wait_duration"] = dbStats.WaitDuration.String()
+	stats["max_idle_closed"] = strconv.FormatInt(dbStats.MaxIdleClosed, 10)
+	stats["max_lifetime_closed"] = strconv.FormatInt(dbStats.MaxLifetimeClosed, 10)
+
+	// Evaluate stats to provide a health message
+	if dbStats.OpenConnections > 90 { // Assuming 100 is the max for this example
```

Comment:
Currently we have `db.SetMaxOpenConns(50)` above so it should never get to 90.

Reply:
> Currently we have `db.SetMaxOpenConns(50)` above so it should never get to 90.

wait let me changed it

Reply:
@briancbarrow Done
I've made the change in commit 90dc1e9. I set the limit to `40` because the maximum is `50`, leaving `10` for any remaining warning messages.